test/mi.c

```diff
@@ -1798,11 +1798,11 @@ static int test_admin_get_log_split_cb(struct nvme_mi_ep *ep,
 		break;
 	case 1:
 		assert(len == 4096);
-		assert(off == 4096);
+		assert(off == 0);
 		break;
 	case 2:
 		assert(len == 4);
-		assert(off == 4096 * 2);
+		assert(off == 0);
```

Comment:
We should probably check the `lpo` field here too, right?

Reply:
Done